pkgs/tools/security/keysmith/default.nix

```
@@ -0,0 +1,33 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, extra-cmake-modules
+, qtbase
+, qtquickcontrols2
+, kirigami2
+, oathToolkit
+}:
+stdenv.mkDerivation rec {
+
+  pname = "Keysmith";
```

generally all lower case
```suggestion
  pname = "keysmith";
```